docs/odk-workflows/UpdateImports.md

```diff
@@ -1,27 +1,28 @@
 # Update Imports Workflow
 
-This page discusses how to update the contents of your imports, like adding or removing terms. If you are looking to customise imports, like changing the module type, see [here](RepoManagement.md).
+This page details the import workflows mostly used by CL editors. 
```

Comment:
This page is automatically generated from ODK. Any changes here will be overwritten every time. I think you already noticed this. The right place to make the changes in the ODK, it's [here](https://github.com/INCATools/ontology-development-kit/blob/a88aaa0bbac89b267f4c1699caa94f32eb53bce4/template/_dynamic_files.jinja2#L1484-L1659). Please, feel free to create a PR or an issue.